src/district/ui/smart_contracts/events.cljs

```diff
@@ -10,18 +10,35 @@
     [district.ui.web3.events :as web3-events]
     [district0x.re-frame.spec-interceptors :refer [validate-first-arg validate-args]]
     [re-frame.core :refer [reg-event-fx trim-v]]
-    [clojure.string :as string]))
+    [clojure.string :as string])
+  (:require-macros [district.ui.smart-contracts.utils :refer [slurp-env-contracts]]))
```

Comment:
you could use the pattern we have in MF for macros, where they are first required in a cljs namespace and then `:required` like normal functions: 
https://github.com/district0x/memefactory/blob/master/src/memefactory/server/macros.clj
https://github.com/district0x/memefactory/blob/master/src/memefactory/server/macros.cljs

Reply:
what is the benefit of doing that when they are not our macros? it is just for using core.async

Reply:
I'm talking about the `utils` macros, which are ours :) The benefit is they can be used from the library like any cljs function, DCE can also work on them

Reply:
I do like the pattern but we are just using these macros to pull in env vars at compile time and I don't see us ever using them beyond this. Please correct me if I'm wrong.